app/code/Tasks/FeedbackManager/Controller/Manage/Delete.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Tasks\FeedbackManager\Controller\Manage;
+
+use Magento\Customer\Controller\AbstractAccount;
+use Magento\Framework\App\Action\Context;
+use Magento\Customer\Model\Session;
+
+class Delete extends \Magento\Framework\App\Action\Action
+{
+    protected $feedbackFactory;
+    protected $customerSession;
+    protected $messageManager;
+    protected $jsonData;
+
+    public function __construct(
+        Context $context,
+        \Tasks\FeedbackManager\Model\FeedbackFactory $feedbackFactory,
+        Session $customerSession,
+        \Magento\Framework\Json\Helper\Data $jsonData
+    ) {
+        $this->feedbackFactory = $feedbackFactory;
+        $this->customerSession = $customerSession;
+        $this->jsonData = $jsonData;
+        parent::__construct($context);
+    }
+
+    public function execute()
```

Comment:
@gayathri-aravind This method can be refactored for better readability

Reply:
Code within the execute method got aligned properly for better readability. Commit: 882571dc9e3988e3e1f109a2d06fe316213a2971